code/drasil-docLang/lib/Drasil/Sections/GeneralSystDesc.hs

```diff
@@ -17,17 +17,17 @@ genSysIntro = foldlSP [S "This", phrase section_, S "provides general",
 
 -- | User Characeristics section constructor. Does not contain any subsections.
 usrCharsF :: [Contents] -> Section
-usrCharsF [] = SRS.userChar [foldlSP[S "There are no", plural userCharacteristic]] []
+usrCharsF [] = SRS.userChar [emptySectSentence userCharacteristic] []
 usrCharsF intro = SRS.userChar intro []
 
 -- | System Constraints section constructor.
 -- Generalized if no constraints, but if there are, they can be passed through.
 systCon :: [Contents] -> [Section] -> Section
 systCon [] subSec  = SRS.sysCon [systCon_none] subSec
-            where systCon_none = mkParagraph (S "There are no" +:+. plural systemConstraint)
+            where systCon_none = emptySectSentence systemConstraint
```

Comment:
Might as well just inline this (e.g., remove the definition of `systCon_none`)

Reply:
I will make that change right now!